tests/model/DbDatetimeTest.php

```diff
@@ -1,145 +1,224 @@
-<?php
+	<?php
+
+use SilverStripe\Model\FieldType\DBField;
+use SilverStripe\Model\FieldType\DBDatetime;
+
+/**
+ * Tests for {@link SS_Datetime} class.
+ *
+ * @todo Current date comparisons are slightly dodgy, as they only compare
+ *  the current date (not hour, minute, second) and assume that the date
+ *  doesn't switch throughout the test execution. This means tests might
+ *  fail when run at 23:59:59.
+ *
+ * @package framework
+ * @subpackage tests
+ */
+class DBDatetimeTest extends SapphireTest {
+	public function testNowWithSystemDate() {
+		$systemDatetime = DBField::create_field('SS_Datetime', date('Y-m-d H:i:s'));
```

Comment:
I suggest to swap out all `SS_Datetime` strings in this class with the non-deprecated `DateTime`